lib/comments.js

```diff
@@ -14,38 +16,41 @@ var generateCommentHTML = function(comment, type) {
 
     switch (type) {
         case "image":
-            return "<div class='comment'>" +
-                "<a href='/users/" + escapedUsername + "'>" + escapedUsername + "</a><br>" 
-                + "<span class='time'>" + comment.postedDate.toUTCString() + "</span><br>"
+            return "<div class=\"comment\">"
+                + "<a href=\"/users/" + escapedUsername + "\">" + escapedUsername + "</a><br>"
+                + "<span class=\"time\">" + comment.postedDate.toUTCString() + "</span><br>"
                 + escapedComment
                 + "</div>";
         case "user":
-            return "<div class='comment'>" +
-                "<a href='/images/" + comment.imageID + "'>" + comment.imageID + "</a><br>"
-                + "<span class='time'>" + comment.postedDate.toUTCString() + "</span><br>"
+            const image = data?.images[comment.imageID];
+            return "<div class=\"comment\">"
+                + "<a aria-label=\"Image Link\" href=\"/images/" + comment.imageID + "\">" 
+                + `<img alt=\"Image\" style=\"width:100px\" src=\"/images/${getImageFilename(image?.id, image?.mimetype)}\">`
```

Comment:
Reminding myself to consider bringing in #300 once it gets merged so that images within the comments section are returned in a lesser quality to save on network bandwidth.

Reply:
Will take care of this in a later PR to avoid dragging out the PR for too long.